ROADMAP.md

```diff
@@ -1,74 +1,42 @@
 # OpenFL Project Roadmap
 
-This document is intended to give users and contributors an idea of OpenFL product team's current priorities, features we plan to incorporate over the short, medium, and long term, and call out opportunities for the community to get involved.
+This document is intended to give users and contributors an idea of the OpenFL product team's current priorities, features we plan to incorporate over the short, medium, and long term, and call out opportunities for the community to get involved.
 
 ## When will this document be updated?
-At a minimum once each product release - which we expect to be on a cadence of every 3-4 months. 
+At a minimum once each product release - which we expect to be on quarterly cadence, with the goal of transitioning to monthly releases in 2025.
```

Comment:
do we want to publicize that we will be moving to monthly releases.

Reply:
Indeed, it may be premature to state it as of now. We can always add it later, when we make the decision to switch to monthly.